### PR TITLE
Add minor bug fixes

### PR DESCRIPTION
**Pull request by spd75**
When running the web scraper this semester to update web, I was getting some errors that were directly related to the code.

Here are the changes I made.

### CODE REVIEW

**Review by alanna-zhou — APPROVED** [review]
:o if it works, it works 

wait until kate to approve before merging

**Review by katesliang — APPROVED** [review]
lgtm!

File: course_parser.py
Comment (katesliang) [inline course_parser.py:49]: does `tag.findNext("span")[0]` work? might be more concise

**Review by katesliang — APPROVED** [review]
lgtm. just one comment:

**Pull request merged**
